docset/winserver2019-ps/defender/New-MpPerformanceRecording.md

```diff
@@ -0,0 +1,66 @@
+---
+description: Use this topic to help manage Windows and Windows Server technologies with Windows PowerShell.
+external help file: MSFT_MpPerformanceRecording.cdxml-help.xml
+Module Name: Defender
+ms.date: 08/26/2021
+online version: https://docs.microsoft.com/powershell/module/defender/new-mpperformancerecording?view=windowsserver2019-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: New-MpPerformanceRecording
+---
+
+# New-MpPerformanceRecording
+
+## SYNOPSIS
+Collects a performance recording of MDAV scans.
+
+## SYNTAX
+
+```
+New-MpPerformanceRecording
+   [-RecordTo <String >]
+```
+
+## DESCRIPTION
+
+The `New-MpPerformanceRecording` cmdlet collects a performance recording of Microsoft Defender Antivirus scans. These performance recordings contain Microsoft-Antimalware-Engine and NT kernel process events and can be analyzed after collection using the [Get-MpPerformanceReport](Get-MpPerformanceReport.md#get-mpperformancereport) cmdlet.
+
+This `New-MpPerformanceRecording` cmdlet provides an insight into problematic files that could cause a degradation in the performance of Microsoft Defender Antivirus. This tool is provided “AS IS”, and is not intended to provide suggestions on exclusions. Exclusions can reduce the level of protection on your endpoints. Exclusions, if any, should be defined with caution.
+
+For more information on the Performance analyzer, see Performance Analyzer docs.
```

Comment:
```suggestion
For more information on the performance analyzer, see the [Performance Analyzer](/windows-hardware/test/wpt/windows-performance-analyzer) topic.
```

Reply:
Partially fixed; to include the relative link.